spec/views/curation_concerns/base/_relationships.html.erb_spec.rb

```diff
@@ -14,10 +18,30 @@
     end
   end
 
-  context "when collections are present" do
-    let(:collection_presenters) { [double(id: '456', title: ['Containing collection', 'foobar'], to_s: 'Containing collection')] }
+  context "when children are not present" do
+    before do
+      render 'curation_concerns/base/relationships', presenter: presenter
+    end
+    it "shows the message" do
+      expect(rendered).to match %r{This Work does not have any related works\.}
+    end
+  end
+
+  context "when parents are not present" do
+    before do
+      render 'curation_concerns/base/relationships', presenter: presenter
+    end
+    it "shows the message" do
+      expect(rendered).to match %r{This Work is not currently a part of any works\.}
+    end
+  end
+
+  context "when collections are present and no parents are present" do
+    let(:collection_presenters) { [collection] }
     let(:page) { Capybara::Node::Simple.new(rendered) }
     before do
+      allow(view).to receive(:contextual_path).and_return("/collections/456")
+      allow(model_col).to receive(:model_name).and_return("Collection")
```

Comment:
Would that return something like look something like `allow(model_col).to receive(:model_name).and_return(double(ActiveModel::Name("collection)))`?